nifi-nar-bundles/nifi-groovyx-bundle/nifi-groovyx-processors/src/test/java/org/apache/nifi/processors/groovyx/ExecuteGroovyScriptTest.java

```diff
@@ -105,7 +105,7 @@ public static void setupBeforeClass() throws Exception {
         Assume.assumeTrue("Test only runs on *nix", !SystemUtils.IS_OS_WINDOWS);
         FileUtils.copyDirectory(new File("src/test/resources"), new File("target/test/resources"));
         //prepare database connection
-        System.setProperty("derby.stream.error.file", "target/derby.log");
+        System.setProperty("derby.stream.error.file", "target" + File.separator + "derby.log");
```

Comment:
According to line 105, this test class does not run on Windows. Is there some other possible cause of `derby.log` being created in the `nifi-groovyx-processors` directory?